mPowerSDK/TasksAndSteps/PhonationControllers/APHPhonationTaskViewController.m

```diff
@@ -106,6 +108,27 @@ - (BOOL)taskViewController:(ORKTaskViewController *) __unused taskViewController
     return YES;
 }
 
+- (ORKStepViewController *)taskViewController:(ORKTaskViewController *)taskViewController
+						viewControllerForStep:(ORKStep *)step {
+	if ([step isKindOfClass:[ORKAudioStep class]]) {
+		ORKAudioStepViewController *stepViewController = [[ORKAudioStepViewController alloc] initWithStep:step];
+		//remove "too loud" red indicator; requires ResearchKit commit 42aa4408dc04b6f856cccf33fc3fb4484ac728c4
+		stepViewController.alertThreshold = 1.f;
+		return stepViewController;
+	}
+
+	return nil;
+}
```

Comment:
Please submit a pull request with this code chunk to remove the red threshold indicator.